Yassine dev (Ajout Pages Webs)

yassine128 opened the pull request:
Ajout de plusieurs pages (Registration, Liste de livres, Page informative sur un livre + commentaire, Page admin, Page Acceuil)

yassine128 committed:
Setup Django

Ajout de views et changement de settings

yassine128 committed:
Ajout des pages webs

Ajout des pages:
- Registration
- home
- Liste de livres
- Page d'un livre (emprunt + commentaire)
- Page administrateur

BBArikL commented:
Je vais résoudre les conflits sur les fichiers, mais en attendant, moi ce que j'aimerai voir serait:
- Pour la recherche: www.domain.com*/search?query=la-recherche OU www.domain.com*/search/id-de-la recherche (exemples https://biblio.laval.ca/search/594ae845-cd9e-4a78-bc4f-0d5ac042c97e ou https://biblio.bdeb.qc.ca/in/fr/search.xhtml?sf=*&bookmark=ea443c30-1999-4fa9-8feb-642a7daba586&fc=&queryid=e61b2189-08eb-46c6-9d7b-1b86bf4c1828&pageNo=1&currentSort=DateOfPublicationSearch_date_DESC&pageSize=20)
- Pour la présentation d'item: www.domain.com/item/item-id
- Profil: www.domain.com/profile OU .../profil (ou même les 2)
- Paramètres : www.domain.com/profile/settings OU /profil/parametres  (ou même les 2)
- Même chose pour register / login , inscription / connexion
- Panier de livres? (/checkout/ ou /panier/) pour confirmer ce que l'on veut louer
-  Page de biblio ? (/library/library-id)

Optionnel:
- Contact
- FAQ
- Recherche avancée ("/search/"  seulement)

Si tu as besoin de rappels sur Django, tu peux voir la démo que j'avais fait https://github.com/BBArikL/Django_Demo

BBArikL reviewed (commented):
Voir commentaire plus haut

yassine128 committed:
Ajout de pages webs

Ajout de profil, librairie, panier, parametres et changement dans la liste d'item et la recherche de livres.

BBArikL commented:
Alright looks good, si on a besoin d'aller plus dans les détails pour le profil utilisateur, on créera une vue spécialement pour cela (comme pour la le login / registration).

BBArikL reviewed (approved):
Approving

BBArikL committed:
Merge branch 'Arik-Dev' into Yassine-Dev
